packages/drivers/file-driver/package.json

```diff
@@ -64,13 +98,14 @@
 			"build:docs": {
 				"dependsOn": [
 					"...",
-					"api-extractor:commonjs"
+					"api-extractor:commonjs",
+					"api-extractor:esnext"
 				],
 				"script": false
 			},
 			"tsc": [
-				"...",
-				"typetests:gen"
+				"typetests:gen",
+				"^tsc"
```

Comment:
for my own knowledge. why did we have to add this?

Reply:
That is an error introduce by 'policy-check:fix'.  I've put it back.  Thank you for noticing!